docs/chapters/cross-multi-modality/index.md

```diff
@@ -0,0 +1,42 @@
+## Multi & Cross Modality
+
+Jina is a data type-agnostic framework. It basically can work with any kind of data. 
```

Comment:
"Jina is a data type-agnostic framework, offering one to work with any type of data and run cross- and multi-modal search flows."

Reply:
"Jina is a data type-agnostic framework, **letting you** work with any type of data and run cross- and multi-modal search **Flows**."

Emphasis to show my adjustments